pkg/bff/server.go

```diff
@@ -227,15 +227,25 @@ func (s *Server) SetURL(url string) {
 }
 
 func (s *Server) setupRoutes() (err error) {
+	var (
+		authenticator gin.HandlerFunc
+		tags          gin.HandlerFunc
+		tracing       gin.HandlerFunc
+		bffTags       map[string]string
+	)
+
 	// Instantiate authentication middleware
-	var authenticator gin.HandlerFunc
 	if authenticator, err = auth.Authenticate(s.conf.Auth0); err != nil {
 		return err
 	}
 
-	var tracing gin.HandlerFunc
+	if s.conf.Sentry.UseSentry() {
+		bffTags = map[string]string{"service": "bff"}
+		tags = sentry.UseTags(bffTags)
+	}
+
 	if s.conf.Sentry.UsePerformanceTracking() {
-		tracing = sentry.TrackPerformance()
+		tracing = sentry.TrackPerformance(bffTags)
```

Comment:
We don't really need the `bff` service tag since it's in a separate project, but I added it for consistency and it acts as a placeholder for any tags we would want to define here.